demo/index.html

```diff
@@ -68,12 +68,11 @@
           border-radius: 5px;
           border-width: 1px;
         }
-        
         button.selected {
           background-color: #b3e5fc;
         }
       </style>
-      <template is="dom-bind">
+      <dom-bind><template is="dom-bind">
         <template is="dom-repeat" items="[[containers]]">
```

Comment:
Yup, Kevin says that template extensions don't need wrappers *if* they're nested in a template that's being managed by Polymer in some way (i.e. the template for a `dom-module` or anywhere in the `dom-*` elements`).